packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -29,8 +35,14 @@ Future<void> main(FutureOr<void> testMain()) async {
 /// files from a sibling repository, `flutter/goldens`.
 ///
 /// This comparator will locally clone the `flutter/goldens` repository into
-/// the `$FLUTTER_ROOT/bin/cache/pkg/goldens` folder, then perform the comparison against
-/// the files therein.
+/// the `$FLUTTER_ROOT/bin/cache/pkg/goldens` folder using the [GoldensClient],
+/// then perform the comparison against the files therein.
+///
+/// For testing across all platforms, the [SkiaGoldClient] is used to upload
+/// widgets for framework-related golden tests and process results. Currently
+/// these tests are designed to be run in CI post-submit.
```

Comment:
Maybe document weather in CI post-submit we do both or only compare against skia?